resources/vdir.rb

```diff
@@ -77,45 +81,45 @@
       Chef::Log.debug(cmd)
       shell_out!(cmd, returns: [0, 42, 183])
     end
-  else
-    Chef::Log.debug("#{new_resource} virtual directory already exists - nothing to do")
   end
 end
 
 action :config do
-  if current_resource.physical_path
-    converge_by "Configured the VDIR - \"#{new_resource}\"" do
-      cmd = "#{appcmd(node)} set vdir \"#{application_identifier}\""
-      converge_if_changed :physical_path do
-        cmd << " /physicalPath:\"#{windows_cleanpath(new_resource.physical_path)}\""
```

Comment:
no need to clean windows path since we clean windows file path in `load_current_value`